src/Raven.Client/Documents/Queries/QueryResult.cs

```diff
@@ -19,30 +20,25 @@ public abstract class QueryResult<TResult, TIncludes> : QueryResultBase<TResult,
         /// <summary>
         /// Gets or sets the total results for this query
         /// </summary>
-        public int TotalResults { get; set; }
-
-        /// <summary>
-        /// Gets or sets the total results as int64 for this query
-        /// </summary>
-        public long LongTotalResults { get; set; }
```

Comment:
in 5.4 I would mark this as obsolete with message that it will be removed in next major version of the product (look for a message like that, we have it already)